main.py

```
@@ -0,0 +1,756 @@
+
+        elif data == 'check_subscription':
+            await check_subscription(query, lang)
+
+        elif data == 'select_plan':
+            await show_purchase_details(query, context, lang, user_id)
+            set_user_state(user_id, 'PURCHASE_DETAILS')
+
+        # FIXED: Add handling for purchase_details button
+        elif data == 'purchase_details':
+            await show_purchase_details(query, context, lang, user_id)
+            set_user_state(user_id, 'PURCHASE_DETAILS')
```

Ruff (0.11.9)

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)